leap_year.rb

```diff
@@ -0,0 +1,14 @@
+puts "Enter starting_year"
+starting_year = gets.to_i 
+puts "Enter ending_year"
+ending_year = gets.to_i
+puts ""
+while starting_year.to_i <= ending_year.to_i
+if starting_year % 4 == 0
+    puts starting_year
+elsif starting_year % 100 == 0
+elsif starting_year % 400 == 0
```

Comment:
For the conditional (if/else) statement, we like to start with edge cases and then other possibilities.
So here the if/else block would be:
```
if starting_year % 400 == 0
    puts "#{starting_year} is leap year."
elsif starting_year % 4 = 0 && starting_yea % 100 != 0 
    puts "#{starting_year} is leap year."
end
```